CMakeLists.txt

```diff
@@ -355,7 +355,7 @@ if(TESTBUILD_OPENCL_PROGRAMS)
       message(STATUS "Found clang compiler - ${CLANG_OPENCL_COMPILER}")
 
       find_path(CLANG_OPENCL_INCLUDE_DIR opencl-c.h
-        HINTS ${LLVM_INSTALL_PREFIX}/lib/clang ${LLVM_INSTALL_PREFIX}/lib64/clang
+        HINTS ${LLVM_INSTALL_PREFIX}/lib/clang ${LLVM_INSTALL_PREFIX}/lib64/clang /usr/lib/clang /usr/lib64/clang
```

Comment:
This is beginning to look quite crowded :)
I'm wondering if there is some other better/proper approach to locate that header, an approach that i missed.

Reply:
@LebedevRI We could default `${LLVM_INSTALL_PREFIX}` to `/usr/` when not set.

Reply:
We certainly could, though that would not help at all given the PR description.
The problem is *not* that `LLVM_INSTALL_PREFIX` is not set, the problem is that clang install prefix does not match the llvm install prefix.